ntbs-ui-tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "ntbs-ui-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "selenium-standalone": "^6.16.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Looks like a placeholder script definition?

Reply:
Yes this was just generated for the selenium-standalone installation - not sure what needs to be in package.json, I assume I could just live scripts blank?

Reply:
In that case not sure it's worth much cleaning this file. I would be tempted to add the node_modules folder for this project into source control...
This isn't something one would usually do, but as we're not likely to intentionally mess with versioning at any point. It would avoid the need for npm-install, and exactly pin versioning of the install (minimising `works on my computer` issues).

Reply:
Um, we will achieve the same version-locking result by checking in the lock file, won't we?
And then we don't have to leave any trail for the unusual decision.
We're not launching these tests with npm (which is what this stub seems to be for), so yeah, go ahead and kill the scripts bit - all we really car about in this file I think is the `dependencies` section.

Reply:
Had a chat with Jan about this, couldn't sell him on adding node_modules to source control. We did both agree that `npm ci` should be preferred over `npm install` (installs from `package-lock.json`).

Additionally could you please remove the carat from the selenium-standalone version in package.json?